.changeset/sixty-pots-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@bigcommerce/catalyst-core": minor
+---
+
+Adds localized data fetching withing the beforeRequest client helper. If information is translated (only possible to update via the Admin GraphQL API) then we will return the translated product data.
```

Comment:
We're changing "only" soon so to remove potential confusion in the future I'd prefer we remove that and link out to the docs for the Admin GraphQL API: https://developer.bigcommerce.com/docs/store-operations/catalog/graphql-admin/product-basic-info 🍹

Reply:
Sounds good, let me update real quick